bnd.bnd

```diff
@@ -0,0 +1,32 @@
+Bundle-Activator: \
```

Comment:
Can you use the annotation based approach (https://virtual.osgiusers.org/2018/10/pure-annotation-driven-dev#bundle-activator)?

Reply:
Thanks for the information.  OSGi 7.0 annotation definitely looks cleaner.   Have to explicitly  declare osgi.annontation 7.0.0 dependency locally to use the feature.  I am volunteer myself to fix this in future when sling-parent includes osgi.annontation 7.0.0 (ie. 36?).

Will do another pull request with the latest change.